src/processor/map/map_simple.cpp

```diff
@@ -31,7 +31,7 @@ static DataPos getOutputPos(const LogicalSimple* logicalSimple) {
 std::unique_ptr<PhysicalOperator> PlanMapper::mapUseDatabase(
     planner::LogicalOperator* logicalOperator) {
     auto useDatabase = logicalOperator->constPtrCast<LogicalUseDatabase>();
-    auto printInfo = std::make_unique<OPPrintInfo>(useDatabase->getExpressionsForPrinting());
+    auto printInfo = std::make_unique<UseDatabasePrintInfo>(useDatabase->getExpressionsForPrinting());
```

Comment:
Eventually, later on in the issue, we plan to remove `getExpressionsForPrinting`. So, to make that later part simpler, find a way to pass the information needed for the print info without calling `getExpressionsForPrinting`.  There is most likely a 'getDBName()` function.